README.md

```diff
@@ -53,3 +53,8 @@ Interesting preview:
 
 ### GraphQL
 * [Records](https://openjdk.java.net/jeps/395)
+
+### resteasy-reactive-jackson
+* [Record Patterns](https://openjdk.java.net/jeps/440)
+* [Pattern Matching for switch](https://openjdk.java.net/jeps/441)
+* [Code Snippets in Java API Documentation](https://openjdk.java.net/jeps/413)
```

Comment:
What about other improvements?

Collection helper classes looked nice, scoped values and reactive mode could be a nice area to explore, if things do not leak through the event loop on vert.x side